### PR TITLE
new Timescale Labs tutorial

### PR DESCRIPTION
**Pull request by CoolAssPuppy**

### CODE REVIEW

File: getting-started/exploring-labs.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Setup
+Now that you’ve been invited to Timescale Labs, you’re ready to work with some data. The first thing to do is to create a new database. Timescale Labs is a PostgreSQL database with Timescale extensions already installed. Timescale Labs automatically provisions and manages a TimescaleDB instance in Amazon Web Services.
```
Comment (svenklemm) [inline getting-started/exploring-labs.md]: shouldn't this be TimescaleDB extension

File: getting-started/exploring-labs.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+tsdb => \COPY conditions FROM weather_small_conditions.csv CSV
+tsdb => \COPY locations FROM weather_small_locations.csv CSV
+```
+
+# Next Steps
+To get the most out of your Timescale Labs experience, follow the [Device Ops tutorial]:/tutorials/other-sample-datasets#in-depth-devices or the [Weather data tutorial]:/tutorials/other-sample-datasets#in-depth-weather. We've designed TimescaleDB with simplicity in mind, and this tutorial will get you up and running quickly.
+
+If you have questions or feedback about Timescale Labs, want to provide feedback, or need help with more advanced setup requirements, please reach out to us [on our community Slack]:https://timescaledb.slack.com/archives/CRG0JJ6AF, where our team is standing by to answer your questions.
+
+Remember, Timescale Labs is in private alpha. Don’t be shy! Tell us what you want to see us build for you next.
```
Comment (solugebefola) [inline getting-started/exploring-labs.md]: 'build for you' seems weird somehow?  Especially for docs.

File: getting-started/exploring-labs.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+```bash
+tsdb => \COPY conditions FROM weather_small_conditions.csv CSV
+tsdb => \COPY locations FROM weather_small_locations.csv CSV
+```
+
+# Next Steps
+To get the most out of your Timescale Labs experience, follow the [Device Ops tutorial]:/tutorials/other-sample-datasets#in-depth-devices or the [Weather data tutorial]:/tutorials/other-sample-datasets#in-depth-weather. We've designed TimescaleDB with simplicity in mind, and this tutorial will get you up and running quickly.
+
+If you have questions or feedback about Timescale Labs, want to provide feedback, or need help with more advanced setup requirements, please reach out to us [on our community Slack]:https://timescaledb.slack.com/archives/CRG0JJ6AF, where our team is standing by to answer your questions.
```
[the remark this answers is not in the document]
Comment (CoolAssPuppy) [reply getting-started/exploring-labs.md]: The extra path in the Slack is to take you directly to the #support-labs channel.

File: getting-started/exploring-labs.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+utility is installed on your command line. Follow the instructions for 
+your platform in order to 
+[setup the psql command-line utility](https://blog.timescale.com/tutorials/how-to-install-psql-on-mac-ubuntu-debian-windows/).
+
+Take note of the hostname and password (blacked out in the screenshot 
+above) for your Timescale Labs instance.
+
+Connecting to your Timescale database is as easy copying and pasting 
+your service URL. Note that the link you copied from the start screen 
+will look different than the one below; example-password, example-host
```
Comment (solugebefola) [inline getting-started/exploring-labs.md]: variables like 'example-password' should be set as inline code with single backticks:
```suggestion
will look different than the one below; `example-password`, `example-host` 
```

File: getting-started/exploring-labs.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+your platform in order to 
+[setup the psql command-line utility](https://blog.timescale.com/tutorials/how-to-install-psql-on-mac-ubuntu-debian-windows/).
+
+Take note of the hostname and password (blacked out in the screenshot 
+above) for your Timescale Labs instance.
+
+Connecting to your Timescale database is as easy copying and pasting 
+your service URL. Note that the link you copied from the start screen 
+will look different than the one below; example-password, example-host 
+and example-port will be replaced with actual values for your instance.
```
Comment (solugebefola) [inline getting-started/exploring-labs.md]:
```suggestion
and `example-port` will be replaced with actual values for your instance.
```

File: getting-started/exploring-labs.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+(representing temperature and humidity data from a variety of locations).
+
+For the Device Ops dataset, you’d first set up your schema:
+```bash
+psql -x "postgres://tsdbadmin:example-password@example-host:example-port/tsdb?sslmode=require" < devices.sql
+```
+
+Enter your password when prompted.
+
+Then connect to your database with psql and copy in data to the appropriate tables:
```
Comment (solugebefola) [inline getting-started/exploring-labs.md]: Should be either: All one paragraph, or a numbered list

File: getting-started/exploring-labs.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+a TimescaleDB instance in Amazon Web Services.
+
+>:WARNING: Timescale Labs is in private alpha. [Request an invite][invite-request] . We will keep this document updated as we add features, but the screenshots and instructions may be temporarily out of date.
+
+## Sign in to Timescale Labs [](signin)
+Sign in to Timescale Labs and you’ll see that a `timescale_demo` 
+service is automatically created for you. Click on the service 
+and you’ll see a screen similar to this:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/labs_config_screen.png" alt="timescale labs config screen"/>
```
Comment (solugebefola) [inline getting-started/exploring-labs.md]: The image was located on `assets.timescale.com` so src wasn't working, but I just uploaded it to iobeam.com as well

**Pull request merged**